pkg/kubelet/pod_workers_test.go

```diff
@@ -54,6 +54,8 @@ type fakePodWorkers struct {
 	removeRuntime         map[types.UID]bool
 	removeContent         map[types.UID]bool
 	terminatingStaticPods map[string]bool
+
+	syncState map[types.UID]PodWorkerState
```

Comment:
Is the idea that `syncState` would be the output of `SyncKnownPods` https://github.com/kubernetes/kubernetes/blob/76694983a1c50532206504e2b5e2b9767851354e/pkg/kubelet/pod_workers.go#L1038 if it were run?

Reply:
Yes, will clarify that.